mkosi

```diff
@@ -457,6 +457,34 @@ def install_arch(args, workspace, run_build_script):
 
     subprocess.run(cmdline, check=True)
 
+    subprocess.run(["arch-chroot", os.path.join(workspace, "root"), "systemctl", "enable", "systemd-networkd"], check=True)
+    subprocess.run(["arch-chroot", os.path.join(workspace, "root"), "systemctl", "enable", "systemd-resolved"], check=True)
```

Comment:
hmpf, I don't like the mixing of different chrooting tools (i.e. nspawn vs arch-chroot), and invoking code from inside the image too much.

I'd feel a lot more comfortable if this would use systemctl from the host with its --root= switch.